flycheck.el

```diff
@@ -7512,6 +7512,20 @@ See URL `https://github.com/eslint/eslint'."
                                    (flycheck-error-message err))))
                           (flycheck-sanitize-errors errors))
                   errors)
+  :enabled (lambda (checker)
+             (let* ((command (concat (flycheck-find-checker-executable checker)
+                                     (if flycheck-eslintrc
```

Comment:
@Simplify Yes, I think we can safely do that in this pull request.  I don't think that anyone still uses that option.